infra/test/integration/reviewsite/serverspec/default_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'serverspec'
+
+USER = "vagrant"
+WORKSPACE = "/home/#{USER}/workspace"
+PATH = "/opt/rbenv/shims:/opt/rbenv/bin:/opt/rbenv/plugins/ruby_build/bin:$PATH";
+BUNDLE_PATH = "~/bundle"
+
+# Required by serverspec
+set :backend, :exec
+set :path, PATH
+
+RSpec.configure do |c|
+  user    = USER
+end
+
+# Ruby 1.9.3-p448
+describe command('rbenv global 1.9.3-p448 && rbenv version') do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.